lib/clipboard.zsh

```diff
@@ -24,7 +24,7 @@ function clipcopy() {
     else
       cat $file | pbcopy
     fi
-  elif [[ $OSTYPE == cygwin* ]]; then
+  elif [[ $OSTYPE == (cygwin*|msys)* ]]; then
```

Comment:
It should read `(cygwin|msys)*`. Same below.

Reply:
Thanks, done.